src/main.rs

```diff
@@ -133,7 +133,7 @@ impl EventHandler for Handler {
 }
 
 #[group]
-#[commands(multiply, ping, quit, dia)]
+#[commands(multiply, ping, quit, dia, owo, uwu, smile, hug, flex, animal, surprise, dance, shrug, flip, unflip, sus, cri, yike, bear)]
```

Comment:
Move the emoji commands into a different group

Reply:
how?

Reply:
add the following beneath the first group
#[group]
#[commands(ALL OF YOUR COMMANDS)]

Reply:
https://docs.rs/serenity/0.8.6/serenity/framework/standard/macros/attr.group.html

Reply:
do i need to do something here too? 
![image](https://user-images.githubusercontent.com/43792441/82133738-5aaded00-97a4-11ea-8cae-55a5a09619d4.png)

Reply:
![image](https://user-images.githubusercontent.com/27538493/82133812-08b99700-97a5-11ea-85f1-b583d257dcbe.png)
![image](https://user-images.githubusercontent.com/27538493/82133815-0f480e80-97a5-11ea-8f7a-a7f6f0270747.png)
Something like that

Reply:
Except emoji